plugins/client.go

```diff
@@ -0,0 +1,217 @@
+package plugins
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/alibaba/pouch/client"
+
+	"github.com/sirupsen/logrus"
+)
+
+var defaultTimeout = 30
+
+// defaultContentTypeis the default content type accepted and sent by the plugins.
+const defaultContentType = "application/vnd.docker.plugins.v1.1+json"
+
+// PluginClient the client which call Plugin ServiceMethod
```

Comment:
Please add every a `.` in every function's comments.

Reply:
Get it